cfme/tests/cloud/test_providers.py

```diff
@@ -827,6 +827,48 @@ def test_domain_id_validation(request, provider):
     assert view.flash.messages[0].type == 'error'
 
 
+@test_requirements.azure
+@pytest.mark.provider([AzureProvider], override=True, selector=ONE)
+def test_vpc_env_selection(setup_provider, provider, appliance, provisioning):
+    """
+    Test selection of components in environment page of cloud instances
+    with selected virtual private cloud
+
+    Polarion:
+        assignee: anikifor
+        casecomponent: WebUI
+        initialEstimate: 1/2h
+        testSteps:
+            1. Provision an Azure Instance from an Image.
+            2. At the environment page, try to select components without vpc with vpc
+        expectedResults:
+            1. Instance provisioned and added successfully
+            2. Items are selected successfully
+
+    Bugzilla:
+        1315945
+    """
+    vm_name = random_vm_name('prov-az')
+    template = provisioning.get('image').get('name')
+    vm = appliance.collections.cloud_instances.instantiate(name=vm_name,
+                                                           provider=provider,
+                                                           template_name=template)
+    # default args select vpc
+    data = vm.vm_default_args
```

Comment:
Question : default args what values for vpc ?
This is not covering test without vpc .

Reply:
There's `cloud_network` and `cloud_subnet` args that we need in this test.
This test needs to cover only "with vpc" as the "without" is covered by provisioning test.


![image](https://user-images.githubusercontent.com/42433123/60083670-6f216b00-9736-11e9-9a8d-60f11477b123.png)